### PR TITLE
Update to 0.2.3 release

### PR DESCRIPTION
**Pull request by ResidentMario**
Bugfix release.

### CODE REVIEW

**Comment by ResidentMario** [comment]
Some issues due to https://github.com/ResidentMario/geoplot/pull/68 being off-by-one requiring a re-release...

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
   name: geoplot
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/g/geoplot/geoplot-{{ version }}.tar.gz
-  sha256: ebd79384c52569349edbeb85c0d7ed0dade8586c90cd8194c6c45d9f18017e37
+  sha256: 0b40d483e4b1b0c612fb3ca0d61e766d9cc75a15cf2d10dd74771fb6f8c8e99d
 
 build:
   number: 1
```
Comment (ocefpaf) [inline recipe/meta.yaml:12]: BTW, next time you can rely on the bot PR, it saves your time and resets the build number (something we usually forget to do ;-p)

Comment (ResidentMario) [reply recipe/meta.yaml:12]: Yeah it's nice that there's a bot for this now! But does the bot pick up new-new versions? I saw it open a PR for `0.2.2`, which I *thought* fixed an issue, but not for `0.2.3`, which *actually* fixed the issue. Perhaps it just didn't have the time however.

Comment (ocefpaf) [reply recipe/meta.yaml:12]: > But does the bot pick up new-new versions? 

Any new release that is on PyPI (the tracked URL for this feedstock) will be picked up.

**Pull request merged**